vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -53,10 +53,12 @@
         "SkipLinuxAzSecPack": "true",
         "os": "Linux",
         "now": "{{user `create_time`}}",
-        "createdBy": "aks-vhd-pipeline"
+        "createdBy": "aks-vhd-pipeline",
+        "image_sku": "{{user `img_sku`}}"
       },
       "location": "{{user `location`}}",
       "vm_size": "{{user `vm_size`}}",
+      "managed_image_storage_account_type": "Premium_LRS",
```

Comment:
Just curious, did we see any perf improvement (like build time decreased?) after changing to Premium_LRS?

Reply:
Yes, it has enabled performance increases throughout the build, particularly when pulling container images. Depending on the VHD it decreases the build time by about 5 minutes by itself. However, it enables the parallelization of pulling container images and kube-proxy/kubernetes binaries to execute much faster, so further optimizations build on the performance that this allows. 

Pulling container images originally took between 9:45 - 17:00. 

With the increased vm size, increased disk tier, and parallelization, it takes between 3:30 - 9:00 minutes the overwhelming majority of the time now.

Reply:
oh, did upstream packer implement support for Premium_LRS? thats cool, it didnt until I guess an year ago?